pkg/jobs/registry.go

```diff
@@ -377,6 +377,25 @@ func (r *Registry) CreateJobWithTxn(ctx context.Context, record Record, txn *kv.
 	return j, nil
 }
 
+const invalidNodeID = 0
+
+// CreateAdoptableJobWithTxn creates a job which will be adopted for execution at a later time
```

Comment:
super minor nit: it looks like this comment and the one below are wrapped to 100 char, rather than the usual 80

Reply:
Done